openedx/core/djangoapps/bookmarks/apps.py

```diff
@@ -32,3 +32,7 @@ class BookmarksConfig(AppConfig):
     def ready(self):
         # Register the signals handled by bookmarks.
         from . import signals
+        # Register the tasks handled by signals.
+        from . import tasks
+        from cms import CELERY_APP
```

Comment:
Could you wrap it with try-except block to handling import error?

Reply:
import of module and setting works correctly

Reply:
Ok, thakyou